### DISCUSSION TITLE
Bulk invite [Jul 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2806] [Discord Threads]
> 🙋‍♂️ **Natalie Weber**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### You can comma separate and invite multiple ppl together  
#  
> 🙋‍♂️ **Natalie Weber**  
#### Is there a way to let them sign themselves up automatically though, and not give them full access to our project?  
#  
> 💁 **o1lab**  
#### ah we'd that before however users didnt to allow signing up and use the projects 😅  
#  
> 💁 **o1lab**  
#### but that was a long time ago  
#  
> 🙋‍♂️ **Natalie Weber**  
#### ah i see. okay, thank you!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/995671686986874964)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
